rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -126,6 +127,10 @@ extern s32 sysutil_send_system_cmd(u64 status, u64 param)
 				return -1;
 			}
 		}
+		else if (status == CELL_SYSUTIL_REQUEST_EXITGAME)
+		{
+			_sys_game_watchdog_stop();
```

Comment:
It doesn't belobg here, it should raise some counter and get checked by watchdor or in the restart callback.

Reply:
Like where?

Reply:
he means, you should just set some flag here and stop the watchdog automatically when it is true.
No need to call a function here.
But to be honest, that function already does nothing but set the flag anyway, so idk why it is necessary to add more code here.

Reply:
> he means, you should just set some flag here and stop the watchdog automatically when it is true. No need to call a function here. But to be honest, that function already does nothing but set the flag anyway, so idk why it is necessary to add more code here.

It turned out that the game didn't call sys_game_watchdog_stop() on it own when CELL_SYSUTIL_REQUEST_EXITGAME was issued, leading to the watchdog's being triggered due to a long GracefulShutdown() wait time.

Reply:
How does it look now?

Reply:
LGTM:
```
·! 0:00:55.195338 GUI: Game window close event issued
·! 0:00:55.195483 sys_game: Aborting LV2 Watchdog Thread...
·! 0:00:55.195512 {LV2 Watchdog Thread} SIG: Thread time: 0.000000s (0.026748Gc); Faults: 0 [rsx:0, spu:0]; [soft:0 hard:0]; Switches:[vol:0 unvol:0]; Wait:[47.351s, spur:0]
```

Reply:
And counter/flag resolves it when it occurs BEFORE watchdog start as well.